packages/webview_flutter_lwe/example/pubspec.yaml

```diff
@@ -3,18 +3,20 @@ description: Demonstrates how to use the webview_flutter_lwe plugin.
 publish_to: "none"
 
 environment:
-  sdk: ">=3.1.0 <4.0.0"
-  flutter: ">=3.13.0"
+  sdk: ^3.5.0
+  flutter: ">=3.24.0"
 
 dependencies:
   flutter:
     sdk: flutter
   path_provider: ^2.0.7
   path_provider_tizen:
     path: ../../path_provider/
-  webview_flutter: ^4.4.2
+  webview_flutter: ^4.10.0
+  webview_flutter_android: ^4.0.0
   webview_flutter_lwe:
     path: ../
+  webview_flutter_wkwebview: ^3.13.0
```

Comment:
webview_flutter_android and webview_flutter_wkwebview are unnecessary.
It seems like they were added because of example/lib/main.dart.
There is no need to keep code that is not related to the our platform.